app/workers/parse_raw_event_job.rb

```diff
@@ -21,6 +21,24 @@ def perform(raw_event_id)
                                       finished_at: meta[:finish],
                                       raw_event_id: re.id)
 
+      position = 0
+      transaction_event = {}
+
+      details.each_with_index do |row, index|
+        if row[:content] == "BEGIN"
+          transaction_event[:type] = row[:type]
+          transaction_event[:start] = row[:start]
+          position = index
+        elsif %W(ROLLBACK COMMIT).include?(row[:content]) && !transaction_event.empty?
```

Comment:
Line is too long. [86/80]<br>Do not use `%W` unless interpolation is needed.  If not, use `%w`.